modules/lvms-download-log-files-and-diagnostics.adoc

```diff
@@ -10,7 +10,7 @@ When {lvms} is unable to automatically resolve a problem, use the must-gather to
 
 * Run the must-gather command from the client connected to {lvms} cluster by running the following command:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ oc adm must-gather --image=registry.redhat.io/odf4/ocs-must-gather-rhel8:v4.12 --dest-dir=<directory-name>
+$ oc adm must-gather --image=registry.redhat.io/lvms4/lvms-must-gather-rhel8:v{product-version} --dest-dir=<directory-name>
```

Comment:
Would a variable version in the command work in the future too? `{product-version}` displays as `Branch Build` in the preview for now, but it will display the current version once merged, so it'll be 4.12 for 4.12, then 4.13 once it GAs, etc. We wouldn't have to manually update it with each release but I want to make sure that this works.

Reply:
Yes, that would work.